tests/e2e/cucumber/steps/api.ts

```diff
@@ -286,3 +286,23 @@ Given(
     }
   }
 )
+
+Given(
+  '{string} creates a public link of following resource using API',
```

Comment:
this should go to the `api.ts`

Reply:
isn't this in api.ts already?

Reply:
my eyes :eyeglasses: . sorry for that :sweat_smile: